linkerd/docs/namer.md

```diff
@@ -354,7 +354,7 @@ host | `marathon.mesos` | The Marathon master host.
 port | `80` | The Marathon master port.
 uriPrefix | none | The Marathon API prefix. This prefix depends on your Marathon configuration. For example, running Marathon locally, the API is avaiable at `localhost:8080/v2/`, while the default setup on AWS/DCOS is `$(dcos config show core.dcos_url)/marathon/v2/apps`.
 ttlMs | `5000` | The polling interval in milliseconds against the marathon API.
-marathonHealth | `false` | Override linkerd's view of instance health with Marathon health checks. In general, setting this to `false` results in more aggressive request-based health detection using [Finagle's Failure Accrual](https://twitter.github.io/finagle/guide/Clients.html#failure-accrual). Setting this to `true` results in more conservative polling of health check endpoints based on [Marathon's health check configuration](https://mesosphere.github.io/marathon/docs/health-checks.html).
+checkMarathonHealth | `false` | Setting this to `false` results in more aggressive request-based health detection using [Finagle's Failure Accrual](https://twitter.github.io/finagle/guide/Clients.html#failure-accrual), ignoring marathon health checks. Setting this to `true` limits linkerd's view of instances to those marathon reports as healthy, based on [Marathon's health check configuration](https://mesosphere.github.io/marathon/docs/health-checks.html). In this case failure accrual still applies to the remaining nodes.
```

Comment:
I'd be interested to hear what other people think but I don't think we need to talk about failure accrual here.  F.A. isn't more aggressive than marathon health checking, it's just a different thing.